app/Models/Payment.php

```
@@ -135,84 +135,85 @@
+            $payment->amount_applied = $request->input('amount_applied');
+            $payment->vat = 0.00;
+            $payment->reference_id = $request->input('reference_id');
+            $payment->remarks = $request->input('remarks');
+            $endTransaction = new EndTransaction();
+            $dateEnd = $endTransaction->getTransactionDate($request->input("branch_id"));
+            $payment->transaction_date = $dateEnd->date_end;
+            $payment->gcash_no = $request->input('gcash_no');
+
+            if ($payment->interest > 0 || $payment->pdi > 0 || $payment->penalty > 0) {
+                $pdi = 0;
+                $penalty = 0;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$penalty'. (Unused Code Rules)

(UnusedLocalVariable)